### PR TITLE
Fix hardcoded framework name

### PR DESCRIPTION
**Pull request by Landerstraeten**

### CODE REVIEW

**Comment by borNfreee** [comment]
While this definitely makes sense, there are several more issues with `PhpUnitPathProvider` class.

Would you mind making it a little bit prettier? :)

1. `PhpUnitPathProvider` should be renamed to something more common, like `TestFrameworkConfigPathProvider` because it is used for any test framework infection run with, not just phpunit

2. There is a `new PhpUnitPathGuesser()` call inside this class that tries to guess PHPUnit's configuration file location based on `composer.json` content (just for Symfony projects at the moment). I think it should be instantiated only when PHPUnit is used as a test framework, but not with PHPSpec

**Comment by Landerstraeten** [comment]
Sure. Will do!

**Comment by Landerstraeten** [comment]
I'll add some additional tests to improve the msi.

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,10 @@ language: php
 sudo: false
 
 php:
-  - 7.0
```
Comment (borNfreee) [inline .travis.yml]: According to http://php.net/supported-versions.php, PHP 7.0 is still maintained version and will get security fixes up to December, 2018.

IMO it's too early to drop it. Currently, there are no PHP 7.1 features that we can't live without. We will just lose many users who are still on PHP7.0.x.

Please revert all the code related to PHP7.1

File: src/Config/ValueProvider/TestFrameworkConfigPathProvider.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        $this->consoleHelper = $consoleHelper;
+        $this->questionHelper = $questionHelper;
+    }
+
+    public function get(InputInterface $input, OutputInterface $output, array $dirsInCurrentDir, string $testFramework): ?string
+    {
+        try {
+            return $this->testFrameworkConfigLocator->locate($testFramework);
+        } catch (\Exception $e) {
+            if ($testFramework !== TestFrameworkTypes::PHPUNIT) {
```
Comment (borNfreee) [inline src/Config/ValueProvider/TestFrameworkConfigPathProvider.php]: just a question: does it make sense to combine this condition with the next one?

```php
if ($testFramework !== TestFrameworkTypes::PHPUNIT || !file_exists('composer.json')) {
   // ...
```

Comment (Landerstraeten) [reply src/Config/ValueProvider/TestFrameworkConfigPathProvider.php]: If you combine them, you need to write additional tests to kill mutants.
And I think it's more readable to just split them and fast return.

**Comment by borNfreee** [comment]
Thank you @Landerstraeten!

**Pull request merged**